packages/ui-default/templates/home_security.html

```
@@ -70,7 +70,7 @@ <h1 class="section__title" id="linked-accounts" data-heading>{{ _('Linked Accoun
           <form method="post">
             <input type="hidden" name="operation" value="unlink_account">
             {%- for relation in relations -%}{% if relation.platform != 'mail' %}
-            {% set platform = handler.loginMethods.find(eval("r => r.id === '" + relation.platform + "'")) %}
+            {% set platform = loginMethods.find(eval("r => r.id === '" + relation.platform + "'")) %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)